autonomy/deploy/base.py

```diff
@@ -90,6 +91,11 @@
     ]
 }
 
+DEFAULT_AGENT_MEMORY_LIMIT = int(os.environ.get("AUTONOMY_AGENT_MEMORY_LIMIT", 1024))
```

Comment:
Note that for kubernetes the default was 1512, is that an issue?

Reply:
I don't think so, event if it is it can be overridden very easily